PhysicsTools/PatAlgos/test/patTuple_addBTagging_cfg.py

```diff
@@ -101,6 +101,17 @@
     ,'pfCombinedCvsBJetTags'
      # ChargeTagging
     ,'pfChargeBJetTags'
+     #Deep Flavour
+    ,'pfDeepCSVJetTags:probb'
+    ,'pfDeepCSVJetTags:probc'
```

Comment:
Well, it is more a "one size fit all" solution, this will work in both, and in the future the Phase 0 checks should be less and less relevant.